src/main/kotlin/net/irisshaders/lilybot/extensions/util/PublicUtilities.kt

```diff
@@ -29,15 +29,23 @@ import net.irisshaders.lilybot.utils.configPresent
 import net.irisshaders.lilybot.utils.responseEmbedInChannel
 import net.irisshaders.lilybot.utils.userDMEmbed
 
+/**
+ * This class contains a few utility commands that can be used by the public in guilds, or that are often seen by the
+ * public.
+ *
+ * @since 3.1.0
+ */
 class PublicUtilities : Extension() {
 	override val name = "public-utilities"
 
 	override suspend fun setup() {
+		// The channel specifically for sending online notifications to
 		val onlineLog = kord.getGuild(TEST_GUILD_ID)?.getChannel(ONLINE_STATUS_CHANNEL) as GuildMessageChannelBehavior
 
 		/**
-		 * Online notification
+		 * Online notification, that is printed to the official [TEST_GUILD_ID]'s [ONLINE_STATUS_CHANNEL].
 		 * @author IMS212
+		 * @since < v 2.0
```

Comment:
Would it not just be 2.0.0 as that was Kotlin version and this doesn't use JDA code?

Reply:
Damn I swear I cleared this